checks.d/splunk_topology.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, instance, init_config):
         })
 
         self.default_polling_interval_seconds = init_config.get('default_polling_interval_seconds', 15)
-
+        self.default_saved_searches_parallel = init_config.get('default_saved_searches_parallel', 3)
```

Comment:
I was under the impression, based on a convo with Kostya, that this was not an issue for events, I haven't looked at it though. I can have a look for events as well and put it in a separate PR.